src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+            }
+        } else if (decision2 == 2) {
+            makeRaiseSmallBlind(player1, player2, amountToCallSmallBlind(player1, player2));
+            preFlopOutPositionPlayerGotRaise(player1, player2);
+        } else {
+            System.out.println(player2.getNamePlayer() + " folds");
+            System.out.println(player1.getNamePlayer() + " wins " + gameState.getPot() + "$");
+        }
+    }
+
+    public void howMuchToCallBigBlind(Player player1, Player player2) {
+        System.out.println(amountToCallBigBlind(player1, player2) + "$ to call");
+    }
+
+    public void howMuchToCallSmallBlind(Player player1, Player player2) {
+        System.out.println(amountToCallSmallBlind(player1, player2) + "$ to call");
+    }
+
+    public double amountToCallBigBlind(Player player1, Player player2) {
+        return player2.getAmountInHand() - player1.getAmountInHand();
+
+    }
+
+    public double amountToCallSmallBlind(Player player1, Player player2) {
+        return player1.getAmountInHand() - player2.getAmountInHand();
+    }
+
+    public void printPot() {
+        System.out.println("Pot wynosi " + gameState.getPot() + "$");
+    }
+
+    public void layCards(GameState gameState) {
+        if (gameState.getGamePhase() == GamePhase.PREFLOP) {
+            makeFlop();
+        } else if (gameState.getGamePhase() == GamePhase.FLOP) {
+            printPot();
+            System.out.println("Turn");
+            makeTurn();
+        } else if (gameState.getGamePhase() == GamePhase.TURN) {
+            printPot();
+            System.out.println("River");
+            makeRiver();
+        } else {
+            // tutaj użyć jaokś resultHandout i winning handa by można i poprawić te printy
+            String whoWins = equityEvaluator.addPlayerWinsTest(result.getBoard(), result.getHand1(), result.getHand2());
+            System.out.print("wins " + whoWins + " " + gameState.getPot() + "$");
+            updatePlayers(whoWins);
+            System.out.println();
+        }
+    }
+
+    public void makeFlop() {
+        System.out.println("Flop");
+        System.out.print(result.getBoard().get(0) + " ");
+        System.out.print(result.getBoard().get(1) + " ");
+        System.out.print(result.getBoard().get(2) + " ");
+        System.out.println();
+    }
+
+    public void makeTurn() {
+        System.out.print(result.getBoard().get(0) + " ");
+        System.out.print(result.getBoard().get(1) + " ");
+        System.out.print(result.getBoard().get(2) + " ");
+        System.out.print(result.getBoard().get(3) + " ");
+        System.out.println();
+    }
+
+    public void makeRiver() {
+        System.out.print(result.getBoard().get(0) + " ");
+        System.out.print(result.getBoard().get(1) + " ");
+        System.out.print(result.getBoard().get(2) + " ");
+        System.out.print(result.getBoard().get(3) + " ");
+        System.out.print(result.getBoard().get(4) + " ");
+        System.out.println();
+    }
```

Comment:
make w nazwie jest slabo pasujac w momencie gdy printujesz cos a nie tworzysz. Ogolnie te 3 funkcje proponuje połączyc w jedna bo robia praktycznie to samo. Zrobic funkcje 
printBoard(GamePhase phase){
System.out.println(phase);
int endIndex = BOARD_PRINT_INDEXES.get(phase) // BOARD_PRINT_INDEXES to hashMapa z intami FLOP:3, TRUN:4, RIVER:5

for(...){
sout
}
}

Reply:
BOARD_PRINT_INDEXES powinnac byc na samej gorze klasy i byc jako public static final Map BOARD_PRINT_INDEXES = ...

Reply:
ogolnie spoko tylko kosmetyczna uwaga do tego jak stworzyles BOARD_PRINT_INDEXES
jest to akceptowalne ale u nas w robocie robi sie cos takiego poprzez 
"    public static final Map<GamePhase, Integer> BOARD_PRINT_INDEXES = Map.of(GamePhase.PREFLOP, 3,
                                                                                    GamePhase.FLOP, 4,
                                                                                    GamePhase.TURN, 5
    );" to wez to zmien bo to jest troche krotsze